deps/build.jl

```diff
@@ -1,15 +1,37 @@
-#
-
-find_llvm() = begin
+#Returns the path as determined by llvm-config, or throws an error if LLVM
+#could not be found
+function find_llvm()
     if haskey(ENV, "TRAVIS")
         return readchomp(`which llvm-config-3.4`)
     end
     tmp = joinpath(JULIA_HOME, "llvm-config")
```

Comment:
I found one problem with the nightly build so far, you can follow along [here](https://github.com/JuliaLang/julia/pull/18047).